src/core/overlay/overlay.view.js

```
@@ -0,0 +1,52 @@
+import View from 'core/view/view';
+
+export default class OverlayView extends View {
+	constructor(model, markup){
+		super();
+
+		this.model = model;
+		this.markup = markup;
+
+		this.onInit();
+	}
+
+	onInit() {
+	}
+
+	position(start, end){
+		if (!this.overlay) {
+			return;
+		}
+
+		this.overlay.style.left = `${Math.min(start.x, end.x)}px`;
+		this.overlay.style.top = `${Math.min(start.y, end.y)}px`;
+		this.overlay.style.width = `${Math.abs(start.x - end.x)}px`;
+		this.overlay.style.height = `${Math.abs(start.y - end.y)}px`;
+	}
+
+	show() {
+		if (this.overlay){
```

maybe we can throw an Exception here?

---

Forget about exception?)